### PR TITLE
UT Georeferencing Survey 1

### PR DESCRIPTION
**Pull request by LeeGregory12d**
# IFC4.3 Unit test

## Unit test
Unit Test of Survey data
There is  a survey string of just one vertex with the string Name "SF LE" and an attribute  "Point Id" with value 3 in a Property Set called Survey. 
There is a survey string of 15 vertices with the string Name "RD ET" but no Pont Id's. The segments between the vertices are straight lines. 

The (x,y) coordinates are in the map projection MGA94 Zone 56 and z is in AHD (Australian Height Datum)

## Review requested
Questions:
1. I've used the entity IfcBuiltElement for the two survey strings but that is just a guess.
   IFCAnnotation mentions survey but that seems strange. Survey strings are fundamental for all infrastructure work, from 
   the initial data such as survey points and breaklines, cadastre,  data, boundaries, control lines, construction points and strings, conformance, machine control etc. 
 So what entities should be used ?

2. All survey strings have names  and in this example they are "SF LE" and "RD ET".
    But survey strings usually have additional information about every vertex on the string. And often additional information for every segment of a survey string.
 In this example I have only one bit of additional information "Point Id" for the one vertex survey string "SF LE" and so I put it in a Property Set called "Survey".
 But what about the more general case when there is more than one vertex in the string and attributes for each vertex and each segment ?
 How should that be coded in Ifc

@pjanck

### CODE REVIEW

File: IFC 4.3/Georeferencing_Survey_1/readme.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+This scenario builds upon following other scenarios:
+- None
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- ifcProjectedCRS
+- ifcMapConversion
```
Comment (pjanck) [inline IFC 4.3/Georeferencing_Survey_1/readme.md:30]: Well, technically, you do have it in the file, but have filled it with default values. So I think that `IfcMapConversion` is not really covered by this unit test.

However, `IfcGeometricCurveSet` is, so please add it.

File: IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc [hunk 2]
```diff
@@ -0,0 +1,74 @@
+ISO-10303-21;
```
Comment (pjanck) [inline IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc:1]: This file only differs from the file above in GUIDs and some semantic data about file's creation date. I don't see the point in having duplicate files.

**Review by pjanck — CHANGES_REQUESTED** [review]
What is the difference between Georeferencing_Survey_1 and Georeferencing_Tiny_Survey_1? I think they are duplicates, so one can be removed.

I get the same geometry in OIP:
grafik

**Comment by pjanck** [comment]
We will address your questions in the next DEPL Tech meeting and provide you with an answer here.

**Comment by LeeGregory12d** [comment]
Stefan,
Because of the initial hassle I had of not knowing about creating a Branch and doing things on Main, I redid my first two examples and submitted them as proper Branches.
So yes those two are the same.

Please keep the second (newer) one “Georeferencing_Survey_1” as it was done as a proper Branch.

Regards
Lee

Dr Lee Gregory
Managing Director
12D Solutions Pty Ltd

Creators of the 12d Model Civil and Surveying Software
and 12d Synergy for Data Management and Project Collaboration.
Ph: +61 (0)2 9970 7117
Email: lee.gregory@12d.com<mailto:lee.gregory@12d.com>
Web: www.12d.com<http://www.12d.com/aus/?utm_source=emailsig&utm_medium=email&utm_campaign=emaillink>

**Comment by pjanck** [comment]
As discussed in the Tech meeting: 

1. Please use `IfcAnnotation` as a semantic object.

2. Can you please provide an example of properties attached to individual segments/points?

**Pull request closed without merging**